great_ape_safe/ape_api/chainlink.py

```diff
@@ -7,12 +7,13 @@ def __init__(self, safe):
         self.safe = safe
 
         # contracts
-        self.relayer = self.safe.contract(
-            r.chainlink.upkeep_registration_requests,
-            interface.IUpkeepRegistrationRequests,
-        )
         self.link = self.safe.contract(r.treasury_tokens.LINK, interface.ILinkToken)
-        self.keeper_registry = self.safe.contract(r.chainlink.keeper_registry)
+        self.keeper_registry = self.safe.contract(
```

Comment:
Not a blocker, just one question. Where is `self.keeper_registry` used? Cannot find any code using this contract

Reply:
this allows for directly calling the keeper registry once the `Chainlink` class has been initialised, eg in these scripts: https://github.com/Badger-Finance/badger-multisig/search?q=keeper_registry